sevntu-checks/pom.xml

```diff
@@ -45,6 +45,17 @@
       <scope>test</scope>
       <version>${checkstyle.eclipse-cs.version}</version>
     </dependency>
+    <!-- required for Jsr305AnnotationsCheck -->
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+      <version>3.0.2</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
```

Comment:
please add ` <!-- required for Jsr305AnnotationsCheck -->` above this too.
It is first time Check bring extra dependencies to library. It is better to keep public notes about this.

Reply:
Done with next commit.